docs/content/docs/tutorials/apigatewayv2-reference-example.md

```diff
@@ -0,0 +1,322 @@
+- AWS IAM permissions to create roles and attach policies to roles.
+- Installed the following tools on the client machine used to access your Kubernetes cluster:
+  - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) - A command line tool for interacting with AWS services.
+  - [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) - A command line tool for working with Kubernetes clusters.
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS clusters.
+  - [Helm 3.7+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+
+### Install the ACK service controller for APIGatewayv2
+
+You can deploy the ACK service controller for Amazon APIGatewayv2 using the [apigatewayv2-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/apigatewayv2-chart).
+You can download it to your workspace using the following command:
+
+```bash
+helm pull oci://public.ecr.aws/aws-controllers-k8s/apigatewayv2-chart --version=v0.0.17
+````
+
+You will need to decompress and extract the Helm chart. You can do so with the following command:
+
+```bash
+tar xzvf apigatewayv2-chart-v0.0.17.tgz
+```
+
+You can now use the Helm chart to deploy the ACK service controller for Amazon APIGatewayv2 to your EKS cluster.
+At a minimum, you need to specify the AWS Region for controller to execute the APIGatewayv2 API calls.
+
+For example, to specify that the APIGatewayv2 API calls go to the `us-east-1` region, you can deploy the service
+controller with the following command:
+
+```bash
+helm install apigatewayv2-chart --generate-name --set=aws.region=us-east-1
+```
+
+For a full list of available values to the Helm chart, please [review the values.yaml file](https://github.com/aws-controllers-k8s/apigatewayv2-controller/blob/main/helm/values.yaml).
+
+### Configure IAM permissions
+
+Once the service controller is deployed, you will need to [configure the IAM permissions][irsa-permissions] for the
+controller to invoke the APIGatewayv2 API. For full details, please review the AWS Controllers for Kubernetes documentation
+for [how to configure the IAM permissions][irsa-permissions]. If you follow the examples in the documentation, use the
+value of `apigatewayv2` for `SERVICE`.
+
+## Create HTTP API
+Execute the following command to create a manifest containing all the APIGatewayv2 custom resources and submit this
+manifest to EKS cluster using kubectl.
+
+{{% hint type="success" title="Referencing Resources" %}}
+Notice that the ACK custom resources reference each other using "*Ref" fields inside the manifest and the user does not
+have to worry about finding APIID, IntegrationID when creating the K8s resource manifests.
+
+You can refer to [API Reference](https://aws-controllers-k8s.github.io/community/reference/) for *APIGatewayv2*
+to find the supported reference fields.
+{{% /hint %}}
+
+```bash
```

Comment:
I will add the manifest into example repo too. For the example repo, I will add raw static manifest.

I was also debating between raw static manifest or using the variables in the manifest. I chose variables because it is consistent with the other tutorials and also allow following commands to reuse the variable.